src/python/pants/backend/go/pkg.py

```diff
@@ -294,5 +430,35 @@ async def resolve_external_module_to_go_packages(
     return ResolveExternalGoModuleToPackagesResult(packages=FrozenOrderedSet(packages))
 
 
+class GoPkgDebugSubsystem(GoalSubsystem):
+    name = "go-pkg-debug"
```

Comment:
Is this temporary?

Reply:
Yes. It allows be to force package metadata resolution so I can inspect it. Will add a comment that it is temporary.